spec.html

```diff
@@ -32339,6 +32355,23 @@ <h1>Math.floor ( _x_ )</h1>
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-math.f16round">
```

Comment:
This puts `f16round` between `floor` and `fround`, which isn't lexicographic order.

ASCII order would put digits before letters (`f16round` before `floor`), but the few examples where it makes a difference in the spec collate digits *after* letters (`atanh` before `atan2`, `log1p` before `log10`), suggesting that `f16round` should be after `fround`.

Reply:
As long as it's adjacent to `fround`, either side is fine by me.